types/react-native/index.d.ts

```diff
@@ -8073,6 +8073,26 @@ export interface ImageStoreStatic {
 export type fetch = (url: string, options?: Object) => Promise<any>
 export const fetch: fetch;
 
+interface FormDataHeaders {
+    [name: string]: string
+};
+
+interface FormDataPart1 {
```

Comment:
The use of union type is possible with Flow but not with Typescript interfaces, isn't it ?